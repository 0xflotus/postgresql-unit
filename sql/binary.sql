SELECT unit_send('0');
SELECT unit_send(-'0'::unit);
SELECT unit_send('1 m'::unit);
SELECT unit_send('1 kg^-1'::unit);
SELECT unit_send('1 s'::unit);
SELECT unit_send('1 A^-1'::unit);
SELECT unit_send('1 K'::unit);
SELECT unit_send('1 mol^-1'::unit);
SELECT unit_send('1 cd'::unit);
SELECT unit_send('1 B^-1'::unit);
