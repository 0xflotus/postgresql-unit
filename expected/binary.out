SELECT unit_send('0');
             unit_send              
------------------------------------
 \x00000000000000000000000000000000
(1 row)

SELECT unit_send(-'0'::unit);
             unit_send              
------------------------------------
 \x80000000000000000000000000000000
(1 row)

SELECT unit_send('1 m'::unit);
             unit_send              
------------------------------------
 \x3ff00000000000000100000000000000
(1 row)

SELECT unit_send('1 kg^-1'::unit);
             unit_send              
------------------------------------
 \x3ff000000000000000ff000000000000
(1 row)

SELECT unit_send('1 s'::unit);
             unit_send              
------------------------------------
 \x3ff00000000000000000010000000000
(1 row)

SELECT unit_send('1 A^-1'::unit);
             unit_send              
------------------------------------
 \x3ff0000000000000000000ff00000000
(1 row)

SELECT unit_send('1 K'::unit);
             unit_send              
------------------------------------
 \x3ff00000000000000000000001000000
(1 row)

SELECT unit_send('1 mol^-1'::unit);
             unit_send              
------------------------------------
 \x3ff00000000000000000000000ff0000
(1 row)

SELECT unit_send('1 cd'::unit);
             unit_send              
------------------------------------
 \x3ff00000000000000000000000000100
(1 row)

SELECT unit_send('1 B^-1'::unit);
             unit_send              
------------------------------------
 \x3ff000000000000000000000000000ff
(1 row)

